access/src/main/external/static/js/admin/src/MoveActionMonitor.js

```diff
@@ -0,0 +1,250 @@
+define('MoveActionMonitor', [ 'jquery'], function($) {
+
+	var defaultOptions = {
+		updateInterval : 5000
+	};
+	
+	function MoveActionMonitor(alertHandler, options) {
+		this.options = $.extend({}, defaultOptions, options);
+		
+		this.alerts = alertHandler;
+		this.moveData = {};
+	}
+	
+	function setDifference(A, B) {
```

Comment:
Why are arrays used to represent sets here? Why not use objects with move ids as the keys?